app/modules/rfswitch.c

```diff
@@ -76,19 +98,25 @@ void transmit(HighLow pulses, bool invertedSignal, int pulseLength, int pin) {
  */
 void send(unsigned long protocol_id, unsigned long pulse_length, unsigned long repeat, unsigned long pin, unsigned long value, unsigned int length) {
   platform_gpio_mode(pin, PLATFORM_GPIO_OUTPUT, PLATFORM_GPIO_FLOAT);
-  Protocol p = proto[protocol_id-1];
+  Protocol p = custom;
+  if(protocol_id > 0) proto[protocol_id-1];
```

Comment:
I think that an assignement to p is missing here.

Reply:
yes this is an oversight, i'll fix and update. thanks!